src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -68,6 +75,16 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives
 
   val trialProjectManager = system.actorOf(ProjectManager.props(app.rawlsDAO, app.trialDAO, app.googleServicesDAO), "trial-project-manager")
 
+  if (logitMetricsEnabled) {
+    logger.info("Logit metrics are enabled.")
+    val metricsActor = system.actorOf(MetricsActor.props(app), "metrics-actor")
+    // use a randomized startup delay to avoid multiple instances of this app executing on the same cycle
```

Comment:
If I understand correctly, each instance would be querying across a time range of one second on a 6-hour schedule, so we would be getting two data points every 6 hours under the current configuration.

Reply:
that's correct. I have a random delay on each schedule so they don't hit rawls at the same time ... but I left the schedule on both prod instances because:
1. coordinating across instances is a lot of work
2. having both instances send metrics keeps us safe in the case where we need to take one instance down for maintenance/upgrades/etc
3. recording 8 metrics per day vs. 4 per day is trivial given the size of other logs we send to kibana